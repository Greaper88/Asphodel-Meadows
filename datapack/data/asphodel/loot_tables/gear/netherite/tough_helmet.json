{
    "pools": [
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "name": "minecraft:netherite_helmet",
                    "functions": [
                        {
                            "function": "set_nbt",
                            "tag": "{CustomModelData:113004,HideFlags:2,am_ItemID:108b,am_ToughSet:1b}"
                        },
                        {
                            "function": "set_attributes",
                            "modifiers": [
                                {
                                    "attribute": "generic.armor",
                                    "amount" : 6,
                                    "operation": "addition",
                                    "slot": "head",
                                    "name": "Armor"
                                },
                                {
                                    "attribute": "generic.knockback_resistance",
                                    "amount" : 0.1,
                                    "operation": "addition",
                                    "slot": "head",
                                    "name": "Knockback"
                                },
                                {
                                    "attribute": "generic.armor_toughness",
                                    "amount" : 3,
                                    "operation": "addition",
                                    "slot": "head",
                                    "name": "Toughness"
                                }
                            ]
                        },
                        {
                            "function": "set_name",
                            "name" : 
                            {
                                "text": "Infused Helmet",
                                "color": "gold",
                                "italic": false
                            }
                        },
                        {
                            "function": "set_lore",
                            "lore" : 
                            [
                                {
                                    "text":"[Gear]",
                                    "color":"dark_aqua",
                                    "italic":false
                                },
                                {
                                    "text":" "
                                },
                                {
                                    "text":"A Powerful Helmet, infused",
                                    "color":"gray",
                                    "italic":false
                                },
                                {
                                    "text":"with magical properties.",
                                    "color":"gray",
                                    "italic":false
                                },
                                {
                                    "text":" "
                                },
                                [
                                    {
                                        "text":"Active Essence: [",
                                        "color":"dark_aqua",
                                        "italic":false
                                    },
                                    {
                                        "text":"Armored",
                                        "color":"blue",
                                        "italic":false
                                    },
                                    {
                                        "text":"]",
                                        "color":"dark_aqua",
                                        "italic":false
                                    }

                                ],
                                {
                                    "text":" "
                                },
                                {
                                    "text":"When on head:",
                                    "color":"gray",
                                    "italic":false
                                },
                                {
                                    "text":"+6 Armor",
                                    "color":"blue",
                                    "italic":false
                                },
                                {
                                    "text":"+3 Armor Toughness",
                                    "color":"blue",
                                    "italic":false
                                },
                                {
                                    "text":"+1 Knockback Resistance",
                                    "color":"blue",
                                    "italic":false
                                },
                                {
                                    "text":" "
                                },
                                {
                                    "text":"  ❂ Asphodel Meadows ❂  ",
                                    "color":"dark_gray",
                                    "italic":false
                                }
                            ]
                        }
                    ]
                }
            ]
        }
    ]
}