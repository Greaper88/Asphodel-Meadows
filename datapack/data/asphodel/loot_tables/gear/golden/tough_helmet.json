{
    "pools": [
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "name": "minecraft:golden_helmet",
                    "functions": [
                        {
                            "function": "set_nbt",
                            "tag": "{CustomModelData:113004,HideFlags:2,am_ItemID:60b,am_ToughSet:1b}"
                        },
                        {
                            "function": "set_attributes",
                            "modifiers": [
                                {
                                    "attribute": "generic.armor",
                                    "amount" : 4,
                                    "operation": "addition",
                                    "slot": "head",
                                    "name": "Armor"
                                }
                            ]
                        },
                        {
                            "function": "set_name",
                            "name" : 
                            {
                                "text": "Infused Helmet",
                                "color": "gold",
                                "italic": false
                            }
                        },
                        {
                            "function": "set_lore",
                            "lore" : 
                            [
                                {
                                    "text":"[Gear]",
                                    "color":"dark_aqua",
                                    "italic":false
                                },
                                {
                                    "text":" "
                                },
                                {
                                    "text":"A Powerful Helmet, infused",
                                    "color":"gray",
                                    "italic":false
                                },
                                {
                                    "text":"with magical properties.",
                                    "color":"gray",
                                    "italic":false
                                },
                                {
                                    "text":" "
                                },
                                [
                                    {
                                        "text":"Active Essence: [",
                                        "color":"dark_aqua",
                                        "italic":false
                                    },
                                    {
                                        "text":"Armored",
                                        "color":"blue",
                                        "italic":false
                                    },
                                    {
                                        "text":"]",
                                        "color":"dark_aqua",
                                        "italic":false
                                    }

                                ],
                                {
                                    "text":" "
                                },
                                {
                                    "text":"When on head:",
                                    "color":"gray",
                                    "italic":false
                                },
                                {
                                    "text":"+4 Armor",
                                    "color":"blue",
                                    "italic":false
                                },
                                {
                                    "text":" "
                                },
                                {
                                    "text":"  ❂ Asphodel Meadows ❂  ",
                                    "color":"dark_gray",
                                    "italic":false
                                }
                            ]
                        }
                    ]
                }
            ]
        }
    ]
}